packages/os/corndog-toml

```diff
@@ -0,0 +1,21 @@
+[required-extensions]
+std = { version = "v1", helpers = ["default"]}
+kernel = "v1"
++++
+lockdown = "{{settings.kernel.lockdown}}"
+
+{{#if settings.kernel.sysctl}}
+[sysctl]
+{{#each settings.kernel.sysctl}}
+"{{@key}}" = "{{this}}"
+{{/each}}
+{{/if}}
+
+{{#if settings.kernel.modules}}
+{{#each settings.kernel.modules}}
+[modules."{{@key}}"]
+allowed = {{this.allowed}}
+autoload = {{this.autoload}}
+{{/each}}
+{{/if}}
+
```

Comment:
There's an extra newline at the end of the file.

VS Code has an "insert final newline" option and other editors will have something similar - it shouldn't be something you have to do manually.